src/components/PDOKAutoSuggest/__tests__/PDOKAutoSuggest.test.js

```diff
@@ -8,13 +8,35 @@ import { formatPDOKResponse } from 'shared/services/map-location';
 import PDOKAutoSuggest, { formatResponseFunc } from '..';
 
 const mockResponse = JSON.stringify(JSONResponse);
-fetch.mockResponse(mockResponse);
 
 const onSelect = jest.fn();
 const resolveAfterMs = timeMs => new Promise(resolve => setTimeout(resolve, timeMs));
+const municipalityQs = 'fq=gemeentenaam:';
+const fieldListQs = 'fl=';
+const defaultFieldsQs = 'id,weergavenaam';
+
+const renderAndSearch = async (value = 'Dam', props = {}) => {
+  const result = render(withAppContext(<PDOKAutoSuggest onSelect={onSelect} {...props} />));
+  const input = result.container.querySelector('input[aria-autocomplete]');
+
+  input.focus();
+
+  act(() => {
+    fireEvent.change(input, { target: { value } });
+  });
+
+  await wait(() => resolveAfterMs(INPUT_DELAY));
```

Comment:
`wait` Is deprecated in the upcoming version of `testing-library`. It's been replaced by `waitFor`.

Reply:
Replaced it. Had to update `@testing-library/react`. Removed `@testing-library/dom` since it's a dependency of `@testing-library/react` and not used anywhere directly.